app/views/layouts/application.html.erb

```diff
@@ -8,21 +8,20 @@
     <div class = 'Navbar'>
 
       <%= link_to "Home Page", root_path %>
+      <%= link_to " Posts " , posts_path %>
```

Comment:
Not sure if this matters too much but to keep it consistent with the other it may be good to remove the whitespace in " Posts ".

Reply:
I added it for my own benefit as other links were too close to it, you're definitely right though. I will look at styling it with CSS at a later stage to separate out the links.